src/containers/sign-on/store/sagas.js

```diff
@@ -112,6 +114,20 @@ function* fetchFollowArtistGenre(followArtistCategory) {
   }
 }
 
+function* fetchReferrer(action) {
+  const { location } = action
+  const handle = new URLSearchParams(location.search).get('ref')
```

Comment:
Not sure if you would need a url decode here?

Reply:
the other place we do handle param work is with matchPath and that doens't call a uri decode https://github.com/ReactTraining/react-router/blob/42933fe141819e4662113ab2c320bf86be3490fb/packages/react-router/modules/matchPath.js#L28

so i think we're fine?